src/test/java/com/klarna/hiverunner/examples/HelloHiveRunnerParamaterized.java

```diff
@@ -15,8 +15,8 @@
  */
 package com.klarna.hiverunner.examples;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
```

Comment:
This test needs to be renamed to `HelloHiveRunnerParamaterizedTest` (it's quite likely it's not currently being run...)